aws/data_source_aws_arn_test.go

```diff
@@ -41,9 +42,11 @@ func testAccDataSourceAwsArn(name string) resource.TestCheckFunc {
 	}
 }
 
-//lintignore:AWSAT003,AWSAT005
-const testAccDataSourceAwsArnConfig = `
+//lintignore:AWSAT005
+func testAccDataSourceAwsArnConfig(region string) string {
+	return fmt.Sprintf(`
 data "aws_arn" "test" {
-  arn = "arn:aws:rds:eu-west-1:123456789012:db:mysql-db"
+  arn = "arn:aws:rds:%s:123456789012:db:mysql-db"
```

Comment:
To also resolve AWSAT005, it may be easiest to pass in the whole ARN, which can be created outside `TestCase` via

```go
testARN := arn.ARN{
	AccountId: "123456789012",
	Partition: endpoints.AwsPartitionID,
	Region:    endpoints.EuWest1RegionID,
	Resource:  "db:mysql-db",
	Service:   "rds",
}
```

Passing through via:

```go
Config: testAccDataSourceAwsArnConfig(testARN.String()),
```

Then each `TestCheckFunc` can also reference the same information:

```go
resource.TestCheckResourceAttr(resourceName, "account", testARN.AccountId),
resource.TestCheckResourceAttr(resourceName, "partition", testARN.Partition),
resource.TestCheckResourceAttr(resourceName, "region", testARN.Region),
resource.TestCheckResourceAttr(resourceName, "resource", testARN.Resource),
resource.TestCheckResourceAttr(resourceName, "service", testARN.Service),
```

Reply:
So fresh and clean! 👍